src/indicator/algorithm/calculator/undefinedInputAdapter.js

```diff
@@ -1,54 +0,0 @@
-import d3 from 'd3';
-import slidingWindow from './slidingWindow';
-
-// Indicator algorithms are not designed to accomodate leading 'undefined' value.
-// This adapter adds that functionality by adding a corresponding number
-// of 'undefined' values to the output.
-export default function() {
```

Comment:
I'd prefer we keep this and move it into d3fc-technical-indicator. It's much easier to deal with the results if they're in a consistent *shape*, undefined guards on traversals quickly get unwieldy.